src/reference/asciidoc/file.adoc

```diff
@@ -661,6 +661,9 @@ or `FileWritingMessageHandler.MessageFlushPredicate` implementation.
 The predicates are called for each open file.
 See the java docs for these interfaces for more information.
 
+When using `flushInterval`, the interval starts at the last write - the file is flushed only if it is idle for the interval.
+Starting with _version 4.3.7_, and additional property `flushWhenIdle` can be set to `false`, meaning that the interval starts with the first write to a previously flushed (or new) file.
```

Comment:
Because the flush actually closes the file and removes the state; to do what you suggest would mean separating flushing from closing - we'd need more configuration (e.g. `idleIntervalForClose`) or more logic - flush but don't remove state (unless the flush is because of an idle interval).

I just thought it would just add more complexity for no real reason.